protofire/omen-exchange
# OMN Token Bridge

How to Test:
- Since Omen Token is not deployed i used GEN erc-20 in the background...Get some gen
Deposit:
- Go into Your connection and select deposit
- Then approve Omen Token bridge
- Desposit some amount 
Withdraw:
- Go into your connection modal
- Select withdraw
- Enter some amount 
- Since claim option is not scope of this pr only way to confirm if transaction is successful is to go into omni bridge ui and confirm heres link https://omni.xdaichain.com/history
